adal/devApps/XFormsApp/App.cs

```diff
@@ -52,7 +52,7 @@ protected override void OnStart()
             // Handle when your app starts
             AppCenter.Start("android=3ea693de-dcb4-4b65-a1b5-04ad2f71ada0;" +
                   "uwp={Your UWP App secret here};" +
-                  "ios={Your iOS App secret here}",
+                  "ios={4be201cc-2027-4045-b535-d57de5ff36c7}",
```

Comment:
is this a secret?  should this be checked in?  Since it's in a PR if it is a secret, we'll need to roll it...

Reply:
no necessarily a secret, more of an identifier for our app in appcenter. The actual credentials required to access appcenter is stored securely in our Azure DevOps